guides/doc-Content_Management_Guide/topics/Managing_Content_Views.adoc

```diff
@@ -443,17 +443,18 @@ To resolve package dependencies by default, complete the following steps:
 . In the {Project} web UI, navigate to *Administer* > *Settings* and click the *Content* tab.
 . Locate the *Content View Dependency Solving Default*, and select *Yes*.
 
-You can also set the default level of dependency resolution that you want.
-You can select between adding packages to solve dependencies only if the required package does not exist, or to add the latest packages to resolve the dependency even if the package exists in the repository.
-
-To set the default level of dependency resolution, complete the following steps:
-
-. In the {Project} web UI, navigate to *Administer* > *Settings* and click the *Content* tab.
-. Locate the *Content View Dependency Solving Algorithm* and select one of the following options:
+.Considerations
+. The package dependency resolution feature does not consider packages that are installed on your system independently of the Content View, so `{package-update-project}` is not guaranteed to work.
+. Packages can have conflicting dependencies in a Content View.
+If this is the case you might have dependency issues on the client.
```

Comment:
This part isn't strictly true.  It's natural for the repository to have conflicting dependencies between packages, that's natural and not a problem in and of itself not a problem for the "client".  It's dependency solving for Satellite that might break, and *that* results in potentially not copying things that should be copied, and *that* results in broken client-side dependencies.

Reply:
@sjha4 Can you comment?

Reply:
Yes.  Bringing in too much is almost never an issue, unless it ends up bringing in something that itself has broken dependencies.  You have to think about the Satellite depsolving use case separately from what DNF would do.

Reply:
@dralley: Is there an explanation for which one dep solving might bring in? Is it the newer version, or the 1st package that it finds to satisfy either dependency?

Reply:
The minimum set of packages to satisfy all the dependencies - ignoring conflicts between packages.  In the sense that they literally aren't taken into consideration at all.  The client works out what it can install, Satellite just tries to make sure everything is available regardless of what it decides.

I guess I should rephrase what I said earlier, Satellite doesn't have any problems with "conflicts", but other issues (like not all the dependencies being satisfiable) can cause the "best effort" to fail.

Reply:
How should the wording be changed to meet your requirements?

Reply:
```
Packages could have conflicting dependencies in a CV which is expected.
A -> b 1.1
C -> b 1.2
Where A, B and C are example packages where package A has a dependency on version 1.1 of package B, and Package C has a dependency on Version 1.2 of package B.
Both b 1.1 and b 1.2 will be brought in by dependency solving into the CV to satisfy dependencies of both A and C.
```